### PR TITLE
App Cache In Meetings With Theme Change Handler

### PR DESCRIPTION
**Pull request by Mohammed-MSFT**

### CODE REVIEW

**Review by Pawank-MSFT — COMMENTED** [review]
Correct Images

**Review by Pawank-MSFT — APPROVED** [review]
looks correct, Approving!

**Review by Prithvi-MSFT — APPROVED** [review]
approved

**Review by Wajeed-msft — APPROVED** [review]

**Comment by Mohammed-MSFT** [comment]
Executed Build Pipe Line : https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/6404125440

File: samples/app-cache-meetings/nodejs/Images/AppCacheThemeChange.gif
[the remark this answers is not in the document]
Comment (Mohammed-MSFT) [reply samples/app-cache-meetings/nodejs/Images/AppCacheThemeChange.gif:1]: fixed

**Pull request merged**